tests/foreman/ui/test_bookmark.py

```diff
@@ -0,0 +1,606 @@
+"""Test classes for Bookmark tests"""
+# -*- encoding: utf-8 -*-
+from fauxfactory import gen_string
+from nailgun import entities
+from random import choice
+from robottelo.constants import BOOKMARK_ENTITIES, STRING_TYPES
+from robottelo.decorators import (
+    bz_bug_is_open,
+    skip_if_bug_open,
+    stubbed,
+    tier1,
+    tier2,
+)
+from robottelo.test import UITestCase
+from robottelo.ui.locators import common_locators, locators
+from robottelo.ui.session import Session
+
+
+def gen_random_string(length=10):
```

Comment:
I'm kind of puzzled why you wrote a function for this...
why not use
```python
gen_string(choice(STRING_TYPES), 10)
```
in the code straight away?